citations-api/api/src/java/org/sakaiproject/citation/api/CitationCollectionOrder.java

```diff
@@ -0,0 +1,51 @@
+package org.sakaiproject.citation.api;
+
+/**
+ * Created by nickwilson on 6/23/15.
+ */
+public class CitationCollectionOrder {
+
+	private String collectionId;
+	private int location;
+	private String sectionType;
```

Comment:
In what kind of way?  Enums are lists of constants whereas this class is a POJO that (for example) gets populated dynamcially from a row from CITATION_COLLECTION _ORDER.